UIComponets/ViewController/TextViewVC.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  TextViewVC.swift
+//  UIComponets
+//
+//  Created by Rajan Patel on 28/04/23.
+//
+
+import UIKit
+
+class TextViewVC: BaseViewController {
+
+    // MARK: - IBOutlets
+    @IBOutlet private var txtvAddress: UITextView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        txtvAddress.adjustUITextViewHeight()
+        txtvAddress.delegate = self
+        setPlaceholderInTextView(placeholderText: "Address", textfield: txtvAddress)
+    }
+    
+    // MARK: - FUNCTION
+    override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+//        txtvAddress.resignFirstResponder()
+        view.endEditing(true)
+        print(#function)
+    }
+    
+    // MARK: - Set up
+    private func setPlaceholderInTextView(placeholderText: String,textfield: UITextView) {
+        textfield.text = placeholderText
+        textfield.textColor = .lightGray
+        
+    }
+
+}
+
+
+
+// MARK: - Extensions
+extension TextViewVC: UITextViewDelegate {
+    
+    // MARK: - FUNCTION
```

Comment:
mark should be apply on group of function, we you want to add details on function then use document comment on function